README.md

```diff
@@ -210,7 +210,20 @@ The code below will enable push notifications to show up when you app is running
 
 If your user taps on the notification this will be tracked back to Klaviyo as an "Opened Push" event assuming you have implemented the tracking changes discussed above.
 
+## SDK Data Transfer
 
+As version 1.7.0 the SDK will cache incoming data and flush it back to the Klaviyo API on an interval. As of this writing the interval is based on the network link currently being used by the app. The table below shows the flush interval used for each type of connection:
```

Comment:
I'd change "As version 1.7.0" to "Starting with version 1.7.0"

Reply:
```suggestion
Starting with version 1.7.0, the SDK will cache incoming data and flush it back to the Klaviyo API on an interval. As of this writing the interval is based on the network link currently being used by the app. The table below shows the flush interval used for each type of connection:
```